src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,21 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
 
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length / 2; i++) {
```

Comment:
Instead of dividing the length of the array by 2 in the loop condition, consider defining a constant for the number of random figures to generate, which would improve code readability and maintainability.